### DISCUSSION TITLE
Get blog post URL when generating RSS

### DISCUSSION THREAD

**Discussion by jcubic** [facebook/docusaurus#9939] [Q&A] [answered]
I want to add text like this at the end of RSS entry:

```html
<a href="<URL>">see the rest of the article</a>
```

I want to make sure that everybody knows that this is not the whole article by adding a CTA (call to action).

Is there a way to get the URL of the blog post from `createFeedItems`?

**Comment by slorber** [comment, marked as answer]
You have access to all blog post metadata, cf my answer here:
https://github.com/facebook/docusaurus/discussions/9934#discussioncomment-8745070

```js
createFeedItems: async ({ blogPosts,...params }) => {
    const feedItems = await params.defaultCreateFeedItems({ blogPosts, ...params });
    feedItems.forEach(async (feedItem,index) => {
        const blogPost = blogPosts[index]!;
        const permalink = blogPost.metadata.permalink;
 		const cta = `<a href="${permalink}">see the rest of the article</a>`
        feedItem.content = feedItem.content + cta;
    });
    return feedItems;
}
```

**Reply by jcubic** [reply]
Thanks, didn't notice permalink, was searching for something with url in the name.
